src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -145,4 +155,15 @@ public String getPath() {
         return path.toString();
     }
 
+    /**
+     * Verifies that the storage file exists on the disk.
+     * 
+     * @throws StorageFileNotFoundException if the storage file is missing.
+     */
+    public void checkIfExistsOnDisk() throws StorageFileNotFoundException {
+        if (!Files.exists(path)) {
+            throw new StorageFileNotFoundException("Storage file is missing");
+        }
```

Comment:
So, what happens in this method if does exist. That is, what is the normal control flow?